sonar-wrapper.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+set -e
+
+echo "Building docker image"
+docker pull sonarqube
+echo "Running docker container"
+docker run -dit -p 9000:9000 --name sonar sonarqube:latest
+echo "sleeping 60 sec to let sonar start up"
+sleep "60"
+
+source ./.ci/sonar.sh
+
+#uncomment it to get html report
+#mvn sonar:sonar -Dsonar.analysis.mode=preview -Dsonar.issuesReport.html.enable=true
```

Comment:
we have this comment in `sonar.sh` do debug on local - it is enough.
this comment should be removed.